src/api/config/application.rb

```diff
@@ -109,6 +109,10 @@ class Application < Rails::Application
     config.active_record.cache_versioning = true
     config.active_record.collection_cache_versioning = false
 
+    # Disable partial writes to avoid causing incorrect values
+    # to be inserted when changing the default value of a column.
+    config.active_record.partial_inserts = false
```

Comment:
Hm isn't this the default? :)

Reply:
I guess fixing this is the thing you want to do then

https://github.com/openSUSE/open-build-service/blob/6655095aadacd0557c389828fcc4e8c187ab6f5d/src/api/config/application.rb#L41